setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.1.0
+current_version = 3.2.0
```

Comment:
Please leave the version number unchanged. Bumping version will be done when releasing. We may want to introduce a few changes before that.